ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+> Please label the issue either bug or proposal according to its purpose.
```

Comment:
above this first task. 

`<!-- Thanks for filing an issue! Before hitting the button, please fill in the following template.-->`